src/utils.jl

```diff
@@ -10,6 +10,10 @@ Perform filtering on `img` with kernel `kern` using the FFT algorithm.
 imfilter_fft(img::AbstractArray{T,N}, kern::AbstractArray{K,N}) where {T<:Real,K<:Real,N} =
   imfilter(img, centered(kern), Inner(), Algorithm.FFT())
 
+imfilter_fft(img::AbstractArray{<:AbstractGray}, kern::AbstractArray) = 
```

Comment:
@juliahm Curious to ask, is the multi-threads performance consideration the reason why we restrict `imfilter` to FFT here?

Reply:
No reason to restrict to FFT @johnnychen94 , I remember that in the past we had to choose the algorithm explicitly but that nowadays ImageFiltering.jl chooses an algorithm based on heuristics, is that right?

Please feel free to omit the specific algorithm if you feel there are improvements.